java-client/session/src/main/java/io/deephaven/client/impl/Session.java

```diff
@@ -0,0 +1,58 @@
+package io.deephaven.client.impl;
+
+
+import io.deephaven.qst.table.TableSpec;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * A session represents a client-side connection to a Deephaven server.
+ */
+public interface Session extends AutoCloseable {
+
+    /**
+     * Creates a new export with a {@link io.deephaven.client.impl.ExportRequest.Listener#logging()
+     * logging listener}.
+     *
+     * <p>
+     * Equivalent to {@code export(ExportsRequest.logging(table)).get(0)}.
+     *
+     * @param table the table
+     * @return the export
+     */
+    Export export(TableSpec table);
```

Comment:
The reason I don't like this idea is because it meshes two separate ideas together:

* Session allows the user to explicitly control the listener (I'm even thinking about getting rid of export(TableSpec) so the caller *needs* to be aware of their listener)
* TableOperations hides the listener from the user.

In a separate branch, I've got `TableHandle` that wraps an Export to provide this functionality.